### DISCUSSION TITLE
How do i retain the scroll position for react app?

### DISCUSSION THREAD

**Discussion by ym-prathamesh** [vercel/community#765] [Help] [answered]
I have hosted a react app via vercel, and on multiple refresh the page looses its scroll position and scrolls to top.

**Comment by amyegan** [comment, marked as answer]
Hi @ym-prathamesh. This question seems like it is specific to React and will depend on your particular code. If you share your repo or a minimal reproducible example with the React Community, I think they will be able to get you the best answer https://reactjs.org/community/support.html
